src/librustc/middle/expr_use_visitor.rs

```diff
@@ -1125,7 +1125,7 @@ impl<'d,'t,'tcx,TYPER:mc::Typer<'tcx>> ExprUseVisitor<'d,'t,'tcx,TYPER> {
                         // that case we can adjust the length of the
                         // original vec accordingly, but we'd have to
                         // make trans do the right thing, and it would
-                        // only work for `~` vectors. It seems simpler
+                        // only work for `Vec`s. It seems simpler
```

Comment:
Technically this would be refererring to `Box<[T]>` types, as `Vec` isn't a built-in type that the compiler should know about.